jobs/service-fabrik-broker/templates/bin/post-start.erb

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+	
+#set -e # exit immediately if a simple command exits with a non-zero status
+set -u # report the usage of uninitialized variables
+# source properties file here.
+export IS_HA=<%= p("ha_enabled") %>
+	
+echo "gcp post-start script execution has started"
+	
+	
+export AUTHBASEURL="-authbaseurl=https://accounts.google.com/o/oauth2/token"
```

Comment:
can this be in swtich case ..... for other IaaS this properties won come.
Or can we move to ha helper package. So that before executing we just source it.

Reply:
sure.. modified to run this code segment if the IaaS is gcp and HA is enabled.